modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,199 @@
+= Topics as Iceberg Tables
+:description: Learn how to integrate Redpanda topics with Apache Iceberg.
+:page-context-links: [{"name": "Linux", "to": "manage:topic-iceberg-integration.adoc" } ]
+:page-categories: Management, High Availability, Data Replication, Integration
+
+
+The Redpanda integration with Apache Iceberg allows your topic data to be stored in the cloud in Iceberg's table format. This opens up immediate access to your streaming data for analytical systems such as data warehouses like RedShift, Snowflake, and Clickhouse, and big data processing platforms such as Apache Spark and Flink, without setting up and maintaining additional ETL pipelines. 
```

Comment:
```suggestion
Redpanda's integration with Apache Iceberg allows you to store topic data in the cloud in Iceberg's table format. This makes your streaming data available to analytical systems, such as data warehouses like RedShift, Snowflake, and Clickhouse, and big data processing platforms, such as Apache Spark and Flink, without setting up and maintaining additional ETL pipelines. 
```
Just suggestions...

Reply:
@kbatuigas. This comment just gets round passive voice "to be" and also attempts to split up the longer sentences.